modules/pubmaticBidAdapter.js

```diff
@@ -48,7 +48,8 @@ const VIDEO_CUSTOM_PARAMS = {
   'linearity': DATA_TYPES.NUMBER,
   'placement': DATA_TYPES.NUMBER,
   'minbitrate': DATA_TYPES.NUMBER,
-  'maxbitrate': DATA_TYPES.NUMBER
+  'maxbitrate': DATA_TYPES.NUMBER,
+  'skip': DATA_TYPES.BOOLEAN
```

Comment:
Skip is an integer.

Are you dropping support for 'skippable' ? You may want to check it for robustness or backwards compatibility

Reply:
Both Skip and Skippable are supported, Going forward we can use ortb specific 'skip' parameter instead of video.ext.skippable

Reply:
Okay but it's not a boolean

Reply:
Yes changed it to number